helm/macropower-analytics-panel-server/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ .Values.namespace }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+    {{- if .Values.security.letsencrypt.enabled }}
+    kubernetes.io/tls-acme: "true"
+    {{- end }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  rules:
+  - host: {{ .Values.server.ingress.host }}
+    http:
+      paths:
+      - backend:
+          service:
+            name: {{ include "resource.default.name" . }}
+            port:
+              number: {{ .Values.server.service.port }}
+          path: /{{ .Values.server.ingress.analytics }}(/|$)(.*)
+          pathType: ImplementationSpecific
+  tls:
```

Comment:
Could we make the entire TLS section optional? Because if we manage to get this going on the Grafana domain then it already has TLS managed by the Grafana ingress so we shouldn't mess with it here.

Reply:
Yes you are right :)